packages/paste-core/components/tabs/src/TabList.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box} from '@twilio-paste/box';
+import {TabPrimitiveList} from '@twilio-paste/tabs-primitive';
+import {TabsContext} from './TabsContext';
+
+interface TabListProps {
+  'aria-label': string;
+  disabled?: boolean | undefined;
+  focusable?: boolean | undefined;
+  children: React.ReactNode;
+}
+
+const HorizontalTabList: React.FC = ({children}) => (
+  <Box display="flex" borderBottomWidth="borderWidth20" borderBottomColor="colorBorderLight" borderBottomStyle="solid">
+    {children}
+  </Box>
+);
+
+const VerticalTabList: React.FC = ({children}) => (
+  <Box borderRightWidth="borderWidth20" borderRightColor="colorBorderLight" borderRightStyle="solid" maxWidth="size30">
+    {children}
+  </Box>
+);
+
+const TabList = React.forwardRef<HTMLDivElement, TabListProps>(({children, ...props}, ref) => {
+  const tab = React.useContext(TabsContext);
+  const TabListWrapper = tab.orientation === 'vertical' ? VerticalTabList : HorizontalTabList;
+  return (
+    <TabPrimitiveList {...(tab as any)} {...props} ref={ref}>
```

Comment:
You could set this ` as={TabListWrapper}` if you wanted to remove the extra DOM element, but that would mean safely spreading props on `HorizontalTabList` and `VerticalTabList`

Reply:
I considered that but for the sake of separating the responsibility thought I'd just make it a child for clarity. Do you feel strongly about it?

Reply:
Nah, not in the slightest! I don’t that function from presentation separation at all